### ISSUE TITLE
Did you compare the result of  CTC  loss  and  cross entropy loss ?  

### ISSUE THREAD

**Issue by Jiakui** [roatienza/deep-text-recognition-benchmark#17]
In the paper, you mentioned that, you can either use CTC loss or cross entropy loss, did you compare the performance of the two different function, which one is better?

Thanks!

**Comment by abdur75648** [comment]
@roatienza , first of all thank you for this amazing work.
It would be very helpful if you could kindly clarify this also.

Also, similar to CTC you used an nn.Linear(256, opt.num_class) layer in the end. Is this implementaion the same as using a CTC at the end? I mean can we say it the model is ViT-STR->BiLSTM->CTC ?
